### PR TITLE
Documents missing parameters for user.list and groups.memberships

### PR DESCRIPTION
**Pull request by mjeammet**
# Goal

Hi, my colleague @tut-tuuut recently submitted a PR to search users by email but she forgot to update the docs ! 
I'm also correcting a small overlook around groups.memberships. 

I'm new to swagger and still have a things or two to figure out :
- how to properly reference the email parameters, which is an array (but arrays require items, which seem to reference a schema ? should I make a schema email ?)
- why exporting to .json changed the formatting of the .json file, hence creating MANY differences
Sorry about that :) 

Thank you for a great tool, by the way :)

### CODE REVIEW

**Comment by tommoor** [comment]
> how to properly reference the email parameters, which is an array

An array of strings should be fine, no need to add additional schema items.

> why exporting to .json changed the formatting of the .json file

Where did you export from, the Swagger editor? It could be they changed the output.

File: spec3.yml [hunk 1]
```diff
@@ -2046,6 +2046,10 @@ paths:
                 - $ref: "#/components/schemas/Pagination"
                 - type: object
                   properties:
+                    id: 
+                      type: string
+                      description: Groupd id
```
Comment (tommoor) [inline spec3.yml:2051]:
```suggestion
                      description: Group id
```

File: spec3.yml [hunk 2]
```diff
@@ -2497,6 +2501,10 @@ paths:
                     query:
                       type: string
                       example: jane
+                    emails[]:
+                      type : string
+                      description: Array of emails 
+                      example: jane.crandall@mail.com
```
Comment (tommoor) [inline spec3.yml:2507]: Wouldn't this need to be an array?

Comment (mjeammet) [reply spec3.yml:2507]: Absolutely, I found the correct syntax this morning.

**Comment by mjeammet** [comment]
> > why exporting to .json changed the formatting of the .json file
> 
> Where did you export from, the Swagger editor? It could be they changed the output.

Yes, I used the "convert and save as JSON" option from the Swagger editor.

**Pull request merged**